src/hast2jcr/handlers/block.js

```diff
@@ -2,7 +2,7 @@ import { select, selectAll } from 'hast-util-select';
 import { toString } from 'hast-util-to-string';
 import { toHtml } from 'hast-util-to-html';
 import button, { getType } from './button.js';
-import { encodeHTMLEntities } from '../utils.js';
+import { encodeHTMLEntities, getHandler } from '../utils.js';
 
 function findNameFilterById(componentDefinition, id) {
   let name = null;
```

Comment:
Unrelated but in this function you compare the id (first class name = block name) with the component id from the definition. these are not necessarily the same.

```if (component.id === id) {```

You would have to compare the component name `component.plugins.xwalk.page.template.name` after making it a class name with `toLowerCase().trim().replaceAll("[^a-z0-9-]", "-")`

Reply:
fixed in https://github.com/mhaack/helix-importer/pull/3